README.md

```diff
@@ -28,15 +28,15 @@ Add to your Gemfile in the `:test` group:
 gem 'rspec-html-matchers'
 ```
 
-and somewhere in RSpec configuration:
+Include it in your RSpec configuration:
```

Comment:
I made this a separate sentence, because otherwise it seems like you might not need to use `gem 'rspec-html-matchers'` if you include it directly in your spec